ui/perfherder/alerts/Pagination.jsx

```diff
@@ -16,6 +16,15 @@ class PaginationGroup extends React.Component {
          aren't working correctly (icons aren't visible)
          so they haven't been added */
       <Pagination aria-label={`Page ${page}`}>
+        {pageNums[0] > 1 && (
+          <PaginationItem className="text-info">
+            <PaginationLink
+              className="text-info"
+              first
+              onClick={() => this.navigatePage(1)}
+            />
+          </PaginationItem>
+        )}
         {page > 1 && (
```

Comment:
It'd also be better to rename `page` such that it's clear it's the current page.

Reply:
`currentPage`?